src/content/docs/zh-cn/develop/Debug/index.mdx

```diff
@@ -1,15 +1,148 @@
 ---
 title: 调试
-description: 调试工作流程的技巧和窍门
+sidebar:
+  label: 概述
+  order: 10
+i18nReady: true
```

Comment:
In order to ensure consistency with the original document format, I added this, but I don't know its usefulness or whether it is necessary.